src/trans/gpu/internal/ext_acc.F90

```diff
@@ -328,7 +328,7 @@ subroutine ext_acc_copyout(ptrs, stream)
     enddo
   end subroutine
   subroutine ext_acc_delete(ptrs, stream)
-    use openacc, only: acc_async_sync
+    use openacc
```

Comment:
@samhatfield instead of deleting all those, just add `acc_delete` here. not sure which acc_delete it is trying to link when it is not given here, but it makes sense that it is crashing.

```suggestion
    use openacc, only: acc_async_sync, acc_delete
```

Reply:
Ah nice spot :) Instead of actually studying the code, I mistakenly relied on the compiler to tell me which symbols needed to be imported. Any idea why the compiler doesn't complain when `acc_delete` isn't imported from `openacc`?

Reply:
I am not a fortran standard expert, maybe you would need to add an `implicit none` somehwere to let it complain...  (maybe just inside the subroutine?). So it then assumes there is a acc_delete, and I think this will just link the acc_delete from the C API which is not compatible with acc_delete from the module.

Reply:
The module has a global "implicit none" at line 10, so this should cover all the included subroutines should it not?

Reply:
It should yes.

Reply:
I'll see if I can create a reproducer and submit it to our compiler devs.

Reply:
I thought about it again. implicit none doesn't cover externals, those can still be implicit. Only F18 adds `implicit none (external,type)`. The external spec would cover this problem, but it is not supported in NVHPC.

Reply:
Interesting. So `implicit none` by itself only checks for variables (equivalent to `implicit none (type)`. The compile won't throw an error if an arbitrary routine is called. The symbol will probably not be resolved at link time of course, but in this case it was being incorrectly resolved from the C API.